src/server/webkit/protocol.ts

```diff
@@ -7428,7 +7428,7 @@ the top of the viewport and Y increases as it proceeds towards the bottom of the
       /**
        * Indicates if this type description has been type Function.
        */
-      isFunction: boolean;
+      isFunction: boolean | undefined;
```

Comment:
How come our protocol.ts is changed without browsers.json? It should be rolled together with the browser.

Reply:
oops, this is just a mass replace